tidb-server/main.go

```diff
@@ -154,6 +156,12 @@ func main() {
 	os.Exit(0)
 }
 
+func syncLog() {
+	if err := log.Sync(); err != nil {
+		println("sync log err: ", err)
```

Comment:
if syncing log failed, should we stop the process with an error code? for example `os.Exit(1)`?

Reply:
how about:
```go
err := log.Sync()
terror.MustNil(err)
```

Reply:
Done.